cmd/remote.go

```diff
@@ -113,35 +114,35 @@ file. Specify a VPS name and an IP address.`,
 	},
 }
 
-// bootstrapCmd represents the remote add command
-var bootstrapCmd = &cobra.Command{
-	Use:   "bootstrap [REMOTE]",
-	Short: "Bootstrap the VPS for continuous deployment",
-	Long: `Bootstrap the VPS for continuous deployment.
+// deployInitCmd represents the inertia [REMOTE] init command
+var deployInitCmd = &cobra.Command{
```

Comment:
I am leaving this where it is for now - after v0.0.1, will likely move it anyway as part of #46